src/TTrigger.cpp

```diff
@@ -1408,7 +1408,7 @@ void TTrigger::execute()
 {
     if( mSoundTrigger )
     {
-        mudlet::self()->playSound( mSoundFile );
+        mudlet::self()->playSound( mSoundFile, 100 );
```

Comment:
I should have commented that line (that 100= volume). Something in the editor to also set the sound volume on sounds attached to triggers wouldn't hurt (i'm not familiar with the gui code)